pkg/build/builder/docker.go

```diff
@@ -120,22 +120,23 @@ func (d *DockerBuilder) Build() error {
 }
 
 // checkSourceURI performs a check on the URI associated with the build
-// to make sure that it is live before proceeding with the build.
-func (d *DockerBuilder) checkSourceURI() error {
+// to make sure that it is valid.  It also optionally tests the connection
+// to the source uri.
+func (d *DockerBuilder) checkSourceURI(testConnection bool) error {
 	rawurl := d.build.Spec.Source.Git.URI
 	if !d.git.ValidCloneSpec(rawurl) {
 		return fmt.Errorf("Invalid git source url: %s", rawurl)
 	}
-	if strings.HasPrefix(rawurl, "git://") || strings.HasPrefix(rawurl, "git@") {
+	if strings.HasPrefix(rawurl, "git@") {
```

Comment:
You can safely remove this condition as well, since we already check that in [ValidCloneSpec](https://github.com/openshift/source-to-image/blob/master/pkg/git/git.go#L64).